tests/services/apiService.test.js

```diff
@@ -1651,69 +1532,102 @@ describe("getPendingBoostClaims function", () => {
   });
 });
 
-describe("getQuestBoostClaimParams function", () => {
```

Comment:
why are we removing this test case?

Reply:
this one is not in my file

Reply:
didnt understand this. I think instead of editing the above test case, this test case is being removed